src/messaging/message_bus_base.ts

```diff
@@ -7,12 +7,42 @@ import { MessageBus } from "./message_bus";
 import { Topic } from "../model/topic";
 import { Ingress } from "../model/ingress";
 import { Egress } from "../model/egress";
+import { LabelHandler } from "../model/label_handler";
+import { LabelsHandlerBase } from "./labels_handler/labels_handler_base";
+import { ExternalLabelsHandler } from "./labels_handler/external_process_labels_handler";
+import { Label } from "../model";
+import { bind } from "lodash";
 
 export abstract class MessageBusBase extends MessageBus {
+
+	private readonly labelHandlers: Map<TopicApiIdentifier["uuid"], Array<LabelsHandlerBase>>;
+
 	public constructor(
-		protected readonly storage: DatabaseFactory
+		protected readonly storage: DatabaseFactory,
 	) {
 		super();
+		this.labelHandlers = new Map<TopicApiIdentifier["uuid"], Array<LabelsHandlerBase>>();
+	}
+
+	protected async onInit(): Promise<void> {
+		await this.storage.using(this.initExecutionContext, async (db) => {
+			const labeldHandlersList = await db.listLabelHandlers(this.initExecutionContext);
+			const differentTopicUuids: Set<TopicApiIdentifier["uuid"]> = labeldHandlersList.reduce((acc, val) => {
```

Comment:
I propose to change type of `differentTopicUuids` from `Set` to `Map`, that will provide ability to do not use `labeldHandlersList.filter` in next `for` block.

Reply:
Fix